src/rpc/rpc_impl.rs

```diff
@@ -169,6 +170,47 @@ impl<N: Network, E: Environment> RpcFunctions<N> for RpcImpl<N, E> {
         Ok(self.ledger.read().await.get_block_header(block_height)?)
     }
 
+    async fn get_blocks_mined(&self) -> Result<Value, RpcError> {
```

Comment:
it would be nice to have a corresponding RPC test, even if it's supposed to return `0`